rpcs3/Emu/system_config.h

```diff
@@ -161,6 +161,7 @@ struct cfg_root : cfg::node
 		cfg::_bool multithreaded_rsx{ this, "Multithreaded RSX", false };
 		cfg::_bool relaxed_zcull_sync{ this, "Relaxed ZCULL Sync", false };
 		cfg::_bool enable_3d{ this, "Enable 3D", false };
+		cfg::_enum<stereo_render_mode_options> stereo_render_mode{ this, "3D Display Mode", stereo_render_mode_options::side_by_side };
```

Comment:
I don't understand why you kept the bool...
Just add a first enum value that represents the 2D default

Reply:
i kept the bool because from what i saw, enabling 3D was a bit more than that bool, and i didn't want to step on any toes by removing it

Reply:
It is only confusing to have more than one variable for the same thing. Either it's 2D or one of the 3D values.
Having an extra bool has no benefit and only adds more elements in all related code